### ISSUE TITLE
login to on-premise portal

### ISSUE THREAD

**Issue by HoppeAndreas** [Esri/ago-assistant#272]
is it possible to have a list of portal url's and their OAuth AppID's available for every user ?
Like the select list when a user has previously logged on to a portal, but available for a user at the first logon.
We use two Enterprise Portals on premise and i don't want to send this information to every user...

i know i can modify the package.json, but this is not working as expected as you would have to use the "Login to ArcGIS Online" button to log in to the enterprise portal. And there is only one url i can modify, so i would have to build two version one for every enterprise portal

**Comment by slibby** [comment]
Good idea! We are actively looking at this functionality for a new version of ArcGIS Assistant in the future. As this app is no longer in development in this repository, the best method might be to look at adding a local storage "insertion" somewhere in the app that pushes the URLs and AppIDs into memory during load, just as if the user has entered it in themselves manually.

**Comment by HoppeAndreas** [comment]
ok, i'll have a look at it and see what i can figure out to load the values.

what exactly do you mean by "no longer in development in this repository" ?
not under development any more or not in this repository ?

**Comment by slibby** [comment]
Sorry for the delay on your comment, it is no longer under development in its current form.
